frontend/src/ui/Common.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+import shadow from './assets/shadow.png';
+import { PrimaryButton, SecondaryButton } from './Button';
+
+export const PageContainer = styled.div`
+  max-width: 800px;
+  margin: auto;
+`;
+
+export const Section = styled.section``;
+
+export const Row = styled.div`
+  display: flex;
+`;
+
+export const [StyledPrimary, StyledSecondary] = [
```

Comment:
Can you move these styled buttons and the shadow button into Button/Button.stories? Ideally common is just for things we haven't categorized yet, and since we already have a Button file, let's use that.